server/openapi.go

```diff
@@ -250,7 +240,7 @@ func init() {
 		},
 	}
 
-	openapiSpec.Paths.Paths["/api/collection"] = p3
+	openapiSpec.Paths.Paths["/api/collection/collectionId"] = p3
```

Comment:
make it obvious that "collectionId" is a param, not the explicit url, i.e. /api/collection/<collectionId>